kompose.spec

```diff
@@ -235,20 +234,29 @@ done
 sort -u -o devel.file-list devel.file-list
 %endif
 
+# Why does check need to use buildroot at all? 
 %check
 %if 0%{?with_check} && 0%{?with_unit_test} && 0%{?with_devel}
 %if ! 0%{?with_bundled}
 export GOPATH=%{buildroot}/%{gopath}:%{gopath}
 %else
-export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd)/vendor:%{gopath}
+#export GOPATH=%{buildroot}/%{gopath}:$(pwd):%{gopath}
+export GOPATH=$(pwd):%{gopath}
 %endif
 
+# Not using %gotest as it doesn't allow us to define our own ldflags
+# Since you can only define ldflags once and %gotest and %buildflags
+# both try to define -ldflags
 %if ! 0%{?gotest:1}
 %global gotest go test
 %endif
 
-%gotest %{import_path}/cli/app
-%gotest %{import_path}/pkg/transformer/kubernetes
+# define testflags here. these were taken from script/test-unit
+# we will need to periodically check these for consistency
+%define testflags -cover -coverprofile=cover.out
```

Comment:
I like all the macros be defined on the same place. Does it make sense to put them at the top of the spec file?